src/stack_pr/cli.py

```diff
@@ -948,6 +948,10 @@ def land_pr(e: StackEntry, remote: str, target: str):
         input=pr_body.encode(),
     )
 
+def abandon_pr(e: StackEntry):
+    log(b("Abandoning ") + e.pprint(), level=2)
```

Comment:
`pprint` takes one argument (that controls whether we're printing PR as links in the terminal):
```suggestion
    log(b("Abandoning ") + e.pprint(False), level=2)
```